lms/templates/faq/custom_faq.html

```diff
@@ -0,0 +1,12 @@
+## mako
+<%page expression_filter="h"/>
```

Comment:
This html template should be in philu's theme.

Reply:
We are setting up PhilU theme, from next sprint (which is starting tomorrow) we will make all template/style changes in PhilU theme.